### ISSUE TITLE
Can't merge user_target_xcconfig for pod targets: ["SVGKit", "geos"].

### ISSUE THREAD

**Issue by Belzik** [GEOSwift/GEOSwift#58]
Hi, guys.

In my project two libraries are very necessary - 'SVGKit', 'GeoSwift' (in addition to him goes 'geos').

I have added to my podfile:
pod 'GEOSwift'
pod 'SVGKit', :git => 'https://github.com/SVGKit/SVGKit.git', :branch => '2.x'

And run 'pod install'. But I receive a error.

[!] Can't merge user_target_xcconfig for pod targets: ["SVGKit", "geos"]. Singular build setting CLANG_CXX_LIBRARY has different values.

[!] Can't merge user_target_xcconfig for pod targets: ["SVGKit", "geos"]. Singular build setting CLANG_CXX_LIBRARY has different values.

Why there is a error?
What can I make?
Help, please.

I tried to lower the SVGKit version and then I have established them. But I have lost in functionality of SVGKit and I can't solve objectives with his help.

**Comment by vfn** [comment]
@Belzik the issue is with `geos` pod. It's a little outdated and needs to have the podspec updated to work with `libc++` instead of `libstdc++`.

I'll try to have a look at it in the next couple of days

**Comment by Belzik** [comment]
Good afternoon, @vfn

**Comment by vfn** [comment]
Hi @Belzik, Good arvo!

I'm sorry mate, but I didn't have a chance to look into that. One patch solution you can apply to work around this issue is to modify the build setting.

Add the following to your Podfile and run `pod install`

```
post_install do |installer|
  installer.pods_project.targets.each do |target|
      if target.name == "geos"
          target.build_configurations.each do |config|
              config.build_settings['CLANG_CXX_LIBRARY'] = 'libc++'
          end
      end
  end
end
```

Let me know if that works

**Comment by Belzik** [comment]
Unfortunately, it doesn't work. (
https://yadi.sk/i/0JARFtzr3NbS3f

**Comment by vfn** [comment]
What program did you use to edit the podfile?

Remove the double quotes from `== “geos”` and add single quotes and then try pod install again

**Comment by Belzik** [comment]
Now I used xcode.

**Comment by Belzik** [comment]
Now I used xcode.
https://yadi.sk/i/_YfSRtcZ3NbZrF

After that - I have removed this block of a code. Also it has tried to be played with it settings.
https://yadi.sk/i/vF1QtpVo3NbaAn

**Comment by Belzik** [comment]
My friend, you have ideas?

**Comment by vfn** [comment]
Hi @Belzik I'm sorry, but that will likely need a new release of the geos podspec, and unfortunately I'm not able to do that at the moment.

You could try to create your own geos pod without the `CLANG_CXX_LIBRARY` flag, and add it as a dependency on your pod file, so GEOSwift will use that as well.

**Comment by vfn** [comment]
@Belzik do you get any build errors, or just the cocoapods warnings?

**Comment by vfn** [comment]
@Belzik try to add the code i previously posted on your cocoapods, but instead of `post_install`, change it to `pre_install`

**Comment by Belzik** [comment]
Hi, my friend. 
https://yadi.sk/i/RyPflX9T3Nx9qM

**Comment by macdrevx** [comment]
This should be resolved in geos 3.7.0/GEOSwift 3.0.0 which are now both available.
